### ISSUE TITLE
GeneralizedTime raises ValueError when dealing with fractional seconds

### ISSUE THREAD

**Issue by tadams42** [etingof/pyasn1#126]
In short, this is what happens:

```
Python 3.5.5 (default, Apr 25 2018, 10:05:30)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from pyasn1.type.useful import GeneralizedTime

In [2]: GeneralizedTime('20170904113726.369').asDateTime
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-e31a8ea96779> in <module>()
----> 1 GeneralizedTime('20170904113726.369').asDateTime

...venv/lib/python3.5/site-packages/pyasn1/type/useful.py in asDateTime(self)
    124             raise error.PyAsn1Error('malformed datetime format %s' % self)
    125
--> 126         return dt.replace(microsecond=ms, tzinfo=tzinfo)
    127
    128     @classmethod

ValueError: microsecond must be in 0..999999
```
I believe the problem is a little bit above, line 107:

```python
                ms = int(ms) * 10000
```
It should be 1000, not 10000

**Comment by etingof** [comment]
Fixed in 2cfdcffb666c1b8756aa4d345ef4cac608286c45. Thanks! ;)

**Comment by etingof** [comment]
Fix released in 0.4.3

**Comment by 0x54726F6E** [comment]
Same problem seems to exist with UTCTime.
